docs/src/main/sphinx/language/types.md

```diff
@@ -41,6 +41,22 @@ This type captures boolean values `true` and `false`.
 
 ## Integer
 
+Integer numbers can be expressed as numeric literals in the following formats:
+
+* Decimal integer. Examples are `-7`, `0`, or `3`.
+* Hexadecimal integer composed of `0X` or `0x` and the value. Examples are
+  `0x0A` for decimal `10` or `0x11` for decimal `17`.
+* Octal integer composed of `0O` or `0o` and the value. Examples are `0o40` for
+  decimal `32` or `0o11` for decimal `9`.
+* Binary integer composed of `0B` or `0b` and the value. Examples are `0b1001`
+  for decimal `9` or `0b101010` for decimal `42``.
+
+Underscore characters are ignored within literal values, and can be used to
+increase readability. For example, decimal integer `123_456.789_123` is
+equivalent to `123456.789123`. Preceding underscores are not permitted.
+
+Integers are supported by the following data types with increasing size range.
```

Comment:
Wdyt about expressing min and max values for different integer types using hex notation? It's easier for me to remember that max is just a different number of f's, like 0x7fff for a small int, and min is different number of zeros, ex. -0x800

Reply:
I think thats a good idea .. in addition to the current values 

From 

```
a minimum value of `-2^7` and a maximum value of `2^7 - 1`.
```

Reply:
-0x8000 and 0x7FFF for small int, 2 digits for every byte, so 32 bit integer is 4 bytes and max for it would be 0x7FFFFFFF (8 characters after 0x). So much easier to remember than 2,147,483,647 😂

Reply:
Added now .. hoping I got it right..